### DISCUSSION TITLE
auto submit the answer without user hitting Enter when it is equal some correct answer

### DISCUSSION THREAD

**Discussion by talkohavy** [SBoudrias/Inquirer.js#1510] [Ideas]
As the title suggests,

Can we have a feature added where the user just types in the correct answer, and it gets submitted without the need for hitting Enter?

This is extremely useful on `confirm` type questions, where simply hitting `n` or `Y` would just submit the answer.

**Comment by SBoudrias** [comment]
Hi 👋🏻 

For now my recommendation will be to [create a custom prompt](https://github.com/SBoudrias/Inquirer.js/tree/main/packages/core) implementing this.

I don't think it'd fit nicely with how the default `confirm` prompt handles input validation today.
